README.md

```diff
@@ -21,6 +21,30 @@ It is executed via an init container, acquiring certificates on pod start.
   see [docs](https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/).
   - This will require configuring kubernetes to grant the `SYS_PTRACE` capability to the container.
 
+#### Generating Certificates
+
+To configure the certificate authority you will need to generate a hex encoded access key, a self signed CA certificate and key for it and store these in kubernetes as secrets. To generate a certificate first create a json file with your configuration (changing the values as necessary):
+
+``` json
+{
+  "CN": "Utility Warehouse CA",
+  "key": {
+    "algo": "ecdsa",
+    "size": 521
+  },
+  "ca": {
+    "expiry": "17520h"
```

Comment:
Might be worth calling this out as 2 years, perhaps?

Reply:
Yeah its one of those things that you need to understand not just copy. I will add something for that